components/TransferBanner.vue

```diff
@@ -51,18 +51,18 @@
             :subject
           />
           <p>
-            <span class="font-bold">{{ $t("This action can't be reverse.") }}</span>
-            <span>{{ $t("You will no longer have access to manage this dataset.") }}</span>
+            <span class="font-bold">{{ $t("Cette action est irréversible.") }}</span>
+            <span>{{ $t("Vous n’aurez plus accès à la gestion de ce jeu de données.") }}</span>
           </p>
           <ProducerSelect
-            v-model="to"
-            :label="$t('Search an organization or a user')"
+            v-model="vers"
```

Comment:
v-model change seems wrong?

Reply:
I'm reverting all the `vers` not wanted, I'm testing pages to find mistakes